source/_patterns/00-atoms/references/reference~data.json

```diff
@@ -1,6 +1,6 @@
 {
   "title": "NKX2-5 mutations causative for congenital heart disease retain functionality and are directed to hundreds of targets",
-  "origin": "NCBI Gene Expression Omnibus. Series accession number <a href=\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE44902\">GSE44902</a>",
+  "origin": "NCBI Gene Expression Omnibus. Series accession number (<a href=\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE44902\">GSE44902</a>)",
```

Comment:
Why is this change included?

Reply:
This is due to there are brackets around this item which Chris mentioned